frontend/src/components/FilterForm/FilterForm.tsx

```diff
@@ -13,7 +13,7 @@ function FilterForm() {
     freelance: false,
     fullTime: false,
   });
-  const [experience, setExperience] = React.useState({
+  const [experience] = React.useState({
```

Comment:
If you are removing setExperience, how are you updating the state?

Reply:
This filter form is not updated at my end.. Remya is having updated code of filter form as she done this functionality.